### DISCUSSION TITLE
[BUG] Get-PnPListItem error: "Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host" if the DocLib has too many items.

### DISCUSSION THREAD

**Discussion by amaru96** [pnp/powershell#3659] [General]
### Discussed in https://github.com/pnp/powershell/discussions/3656

<div type='discussions-op-text'>

<sup>Originally posted by **amaru96** December 28, 2023</sup>
If my doc lib has a massive amount of items (millions) the Get-PnPListItems command throws an error: Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host.

This is the command I'm running which works most of the time, but as I said, if the doclib is massive, it errors.

Get-PnPListItem -List $doclib -Fields "FileLeafRef", "FileRef" -PageSize 100</div>

**Comment by kc-va541-web** [comment]
tracking.
